collections/reverse-engineering-tools/index.md

```diff
@@ -0,0 +1,22 @@
+---
+items:
+  - ReFirmLabs/binwalk
+  - codilime/veles
+  - kaitai-io/kaitai_struct
+  - desowin/usbpcap
+  - icsharpcode/ILSpy
+  - radare/radare2
+  - Storyyeller/Krakatau
+  - deathmarine/Luyten
+  - java-decompiler/jd-gui
+  - 0xd4d/de4dot
+  - avast-tl/retdec
+  - yegord/snowman
+  - x64dbg/x64dbg
+  - mitmproxy/mitmproxy
+  - ohjeongwook/DarunGrim
+  - dbeaver/dbeaver
+  - lucasg/Dependencies
+display_name: Tools helping to understand applications and formats internals.
+---
+Get knowledge!
```

Comment:
Could you expand these descriptions a bit? There's not much text here compared to other collections.

Reply:
this text was inspired by https://github.com/github/explore/blob/master/collections/software-development-tools/index.md#L17